integration-tests/ci-visibility/unskippable-test/test-unskippable.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable jsdoc/valid-types, jsdoc/check-tag-names */
 /**
  * @datadog {"unskippable": true}
```

Comment:
@juan-fernandez I guess we support this notation regularly?

JSDoc linter will complain it is not a valid type and I guess if we support this notation, customers could also run into a similar situation where it is difficult to make this value accepted in JSDocEslintPlugin. Could we wrap it in an object notation? That way it would be regular legit syntax.